scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -229,7 +229,13 @@ abstract class FileSource extends SchemedSource with LocalSourceOverride with Hf
    * TODO: consider writing a more in-depth version of this method in [[TimePathedSource]] that looks for
    * TODO: missing days / hours etc.
    */
-  protected def pathIsGood(globPattern: String, conf: Configuration) = FileSource.globHasNonHiddenPaths(globPattern, conf)
+  protected def pathIsGood(globPattern: String, conf: Configuration) = {
+    if (conf.getBoolean("scalding.require_success_file", false)) {
```

Comment:
can we add this setting to `Config.scala` as well? `.setRequireSuccessFile(value: Boolean): Config` or something to make it easy also for people to programmatically set the option?

Reply:
actually, not. Let's leave it as it is so we don't cause any binary incompatibility by cherry picking this for scalding 0.17.4

Reply:
Ack 👍. Adding it.

Reply:
Okay.